<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="5" passed="4">
  <reporter-output>
  </reporter-output>
  <suite name="Automation Test" duration-ms="73470" started-at="2015-10-16T18:59:30Z" finished-at="2015-10-16T19:00:43Z">
    <groups>
    </groups>
    <test name="Home Page" duration-ms="7757" started-at="2015-10-16T18:59:30Z" finished-at="2015-10-16T18:59:37Z">
      <class name="webTest.VerifyHomePage">
        <test-method status="PASS" signature="setUp()[pri:0, instance:webTest.VerifyHomePage@13dd5ec]" name="setUp" is-config="true" duration-ms="6233" started-at="2015-10-16T14:59:30Z" finished-at="2015-10-16T14:59:36Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="testVerifyHomePage()[pri:0, instance:webTest.VerifyHomePage@13dd5ec]" name="testVerifyHomePage" duration-ms="1065" started-at="2015-10-16T14:59:36Z" finished-at="2015-10-16T14:59:37Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testVerifyHomePage -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:webTest.VerifyHomePage@13dd5ec]" name="tearDown" is-config="true" duration-ms="438" started-at="2015-10-16T14:59:37Z" finished-at="2015-10-16T14:59:37Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- webTest.VerifyHomePage -->
    </test> <!-- Home Page -->
    <test name="Add Guest Validation" duration-ms="8156" started-at="2015-10-16T18:59:37Z" finished-at="2015-10-16T18:59:46Z">
      <class name="webTest.AddGuestPageValidation">
        <test-method status="PASS" signature="setUp()[pri:0, instance:webTest.AddGuestPageValidation@4b706125]" name="setUp" is-config="true" duration-ms="5471" started-at="2015-10-16T14:59:37Z" finished-at="2015-10-16T14:59:43Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="testAddGuestPageValidation()[pri:0, instance:webTest.AddGuestPageValidation@4b706125]" name="testAddGuestPageValidation" duration-ms="2371" started-at="2015-10-16T14:59:43Z" finished-at="2015-10-16T14:59:45Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testAddGuestPageValidation -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:webTest.AddGuestPageValidation@4b706125]" name="tearDown" is-config="true" duration-ms="311" started-at="2015-10-16T14:59:45Z" finished-at="2015-10-16T14:59:46Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- webTest.AddGuestPageValidation -->
    </test> <!-- Add Guest Validation -->
    <test name="Add Guest Details" duration-ms="7545" started-at="2015-10-16T18:59:46Z" finished-at="2015-10-16T18:59:53Z">
      <class name="webTest.AddGuestWithValidData">
        <test-method status="PASS" signature="setUp()[pri:0, instance:webTest.AddGuestWithValidData@43fa0859]" name="setUp" is-config="true" duration-ms="4865" started-at="2015-10-16T14:59:46Z" finished-at="2015-10-16T14:59:50Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="testAddGuestWithValidData()[pri:0, instance:webTest.AddGuestWithValidData@43fa0859]" name="testAddGuestWithValidData" duration-ms="2360" started-at="2015-10-16T14:59:50Z" finished-at="2015-10-16T14:59:53Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testAddGuestWithValidData -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:webTest.AddGuestWithValidData@43fa0859]" name="tearDown" is-config="true" duration-ms="316" started-at="2015-10-16T14:59:53Z" finished-at="2015-10-16T14:59:53Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- webTest.AddGuestWithValidData -->
    </test> <!-- Add Guest Details -->
    <test name="Check Guest Validation" duration-ms="10657" started-at="2015-10-16T18:59:53Z" finished-at="2015-10-16T19:00:04Z">
      <class name="webTest.CheckGuestWithNoData">
        <test-method status="PASS" signature="setUp()[pri:0, instance:webTest.CheckGuestWithNoData@4e88151c]" name="setUp" is-config="true" duration-ms="5041" started-at="2015-10-16T14:59:53Z" finished-at="2015-10-16T14:59:58Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="testCheckGuestWithNoData()[pri:0, instance:webTest.CheckGuestWithNoData@4e88151c]" name="testCheckGuestWithNoData" duration-ms="5287" started-at="2015-10-16T14:59:58Z" finished-at="2015-10-16T15:00:03Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testCheckGuestWithNoData -->
        <test-method status="FAIL" signature="tearDown()[pri:0, instance:webTest.CheckGuestWithNoData@4e88151c]" name="tearDown" is-config="true" duration-ms="0" started-at="2015-10-16T15:00:03Z" finished-at="2015-10-16T15:00:03Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[org.junit.ComparisonFailure: expected:<[Please Enter Guest Name]> but was:<[]>org.junit.ComparisonFailure: expected:<[Only Alphabatic Characters Are Allowed]> but was:<[]>org.junit.ComparisonFailure: expected:<[Please Enter Guest Name]> but was:<[]>]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: org.junit.ComparisonFailure: expected:<[Please Enter Guest Name]> but was:<[]>org.junit.ComparisonFailure: expected:<[Only Alphabatic Characters Are Allowed]> but was:<[]>org.junit.ComparisonFailure: expected:<[Please Enter Guest Name]> but was:<[]>
	at org.junit.Assert.fail(Assert.java:88)
	at webTest.CheckGuestWithNoData.tearDown(CheckGuestWithNoData.java:63)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:510)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:211)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:138)
	at org.testng.TestRunner.afterRun(TestRunner.java:1027)
	at org.testng.TestRunner.run(TestRunner.java:627)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1185)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1110)
	at org.testng.TestNG.run(TestNG.java:1018)
	at org.testng.TestNG.privateMain(TestNG.java:1325)
	at org.testng.TestNG.main(TestNG.java:1294)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- webTest.CheckGuestWithNoData -->
    </test> <!-- Check Guest Validation -->
    <test name="Check Guest With Valid Data" duration-ms="39318" started-at="2015-10-16T19:00:04Z" finished-at="2015-10-16T19:00:43Z">
      <class name="webTest.CheckGuestWithValidData">
        <test-method status="PASS" signature="setUp()[pri:0, instance:webTest.CheckGuestWithValidData@53c52191]" name="setUp" is-config="true" duration-ms="4571" started-at="2015-10-16T15:00:04Z" finished-at="2015-10-16T15:00:08Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="testCheckGuestWithValidData()[pri:0, instance:webTest.CheckGuestWithValidData@53c52191]" name="testCheckGuestWithValidData" duration-ms="33428" started-at="2015-10-16T15:00:08Z" finished-at="2015-10-16T15:00:42Z">
          <exception class="org.openqa.selenium.remote.UnreachableBrowserException">
            <message>
              <![CDATA[Error communicating with the remote browser. It may have died.
Build info: version: '2.48.1', revision: 'd80083d', time: '2015-10-08 21:11:00'
System info: host: 'Miracle-PC', ip: '10.20.214.211', os.name: 'Windows 8.1', os.arch: 'amd64', os.version: '6.3', java.version: '1.7.0_79'
Driver info: driver.version: RemoteWebDriver]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.remote.UnreachableBrowserException: Error communicating with the remote browser. It may have died.
Build info: version: '2.48.1', revision: 'd80083d', time: '2015-10-08 21:11:00'
System info: host: 'Miracle-PC', ip: '10.20.214.211', os.name: 'Windows 8.1', os.arch: 'amd64', os.version: '6.3', java.version: '1.7.0_79'
Driver info: driver.version: RemoteWebDriver
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:641)
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:353)
	at org.openqa.selenium.remote.RemoteWebDriver.findElementByXPath(RemoteWebDriver.java:490)
	at org.openqa.selenium.By$ByXPath.findElement(By.java:361)
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:345)
	at webTest.CheckGuestWithValidData.testCheckGuestWithValidData(CheckGuestWithValidData.java:44)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:821)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1131)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:773)
	at org.testng.TestRunner.run(TestRunner.java:623)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1185)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1110)
	at org.testng.TestNG.run(TestNG.java:1018)
	at org.testng.TestNG.privateMain(TestNG.java:1325)
	at org.testng.TestNG.main(TestNG.java:1294)
Caused by: org.apache.http.conn.HttpHostConnectException: Connect to 127.0.0.1:7055 [/127.0.0.1] failed: Connection refused: connect
	at org.apache.http.impl.conn.DefaultHttpClientConnectionOperator.connect(DefaultHttpClientConnectionOperator.java:151)
	at org.apache.http.impl.conn.PoolingHttpClientConnectionManager.connect(PoolingHttpClientConnectionManager.java:353)
	at org.apache.http.impl.execchain.MainClientExec.establishRoute(MainClientExec.java:380)
	at org.apache.http.impl.execchain.MainClientExec.execute(MainClientExec.java:236)
	at org.apache.http.impl.execchain.ProtocolExec.execute(ProtocolExec.java:184)
	at org.apache.http.impl.execchain.RetryExec.execute(RetryExec.java:88)
	at org.apache.http.impl.execchain.RedirectExec.execute(RedirectExec.java:110)
	at org.apache.http.impl.client.InternalHttpClient.doExecute(InternalHttpClient.java:184)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:71)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:55)
	at org.openqa.selenium.remote.internal.ApacheHttpClient.fallBackExecute(ApacheHttpClient.java:161)
	at org.openqa.selenium.remote.internal.ApacheHttpClient.execute(ApacheHttpClient.java:89)
	at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:142)
	at org.openqa.selenium.firefox.internal.NewProfileExtensionConnection.execute(NewProfileExtensionConnection.java:170)
	at org.openqa.selenium.firefox.FirefoxDriver$LazyCommandExecutor.execute(FirefoxDriver.java:380)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:620)
	... 28 more
Caused by: java.net.ConnectException: Connection refused: connect
	at java.net.DualStackPlainSocketImpl.waitForConnect(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:85)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:339)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:200)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:182)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:579)
	at org.apache.http.conn.socket.PlainConnectionSocketFactory.connectSocket(PlainConnectionSocketFactory.java:74)
	at org.apache.http.impl.conn.DefaultHttpClientConnectionOperator.connect(DefaultHttpClientConnectionOperator.java:134)
	... 43 more
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.remote.UnreachableBrowserException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testCheckGuestWithValidData -->
        <test-method status="FAIL" signature="tearDown()[pri:0, instance:webTest.CheckGuestWithValidData@53c52191]" name="tearDown" is-config="true" duration-ms="0" started-at="2015-10-16T15:00:42Z" finished-at="2015-10-16T15:00:42Z">
          <exception class="org.openqa.selenium.remote.UnreachableBrowserException">
            <message>
              <![CDATA[Error communicating with the remote browser. It may have died.
Build info: version: '2.48.1', revision: 'd80083d', time: '2015-10-08 21:11:00'
System info: host: 'Miracle-PC', ip: '10.20.214.211', os.name: 'Windows 8.1', os.arch: 'amd64', os.version: '6.3', java.version: '1.7.0_79'
Driver info: driver.version: RemoteWebDriver]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.remote.UnreachableBrowserException: Error communicating with the remote browser. It may have died.
Build info: version: '2.48.1', revision: 'd80083d', time: '2015-10-08 21:11:00'
System info: host: 'Miracle-PC', ip: '10.20.214.211', os.name: 'Windows 8.1', os.arch: 'amd64', os.version: '6.3', java.version: '1.7.0_79'
Driver info: driver.version: RemoteWebDriver
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:641)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:670)
	at org.openqa.selenium.remote.RemoteWebDriver.quit(RemoteWebDriver.java:516)
	at webTest.CheckGuestWithValidData.tearDown(CheckGuestWithValidData.java:68)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:510)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:211)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:138)
	at org.testng.TestRunner.afterRun(TestRunner.java:1027)
	at org.testng.TestRunner.run(TestRunner.java:627)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1185)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1110)
	at org.testng.TestNG.run(TestNG.java:1018)
	at org.testng.TestNG.privateMain(TestNG.java:1325)
	at org.testng.TestNG.main(TestNG.java:1294)
Caused by: org.apache.http.conn.HttpHostConnectException: Connect to 127.0.0.1:7055 [/127.0.0.1] failed: Connection refused: connect
	at org.apache.http.impl.conn.DefaultHttpClientConnectionOperator.connect(DefaultHttpClientConnectionOperator.java:151)
	at org.apache.http.impl.conn.PoolingHttpClientConnectionManager.connect(PoolingHttpClientConnectionManager.java:353)
	at org.apache.http.impl.execchain.MainClientExec.establishRoute(MainClientExec.java:380)
	at org.apache.http.impl.execchain.MainClientExec.execute(MainClientExec.java:236)
	at org.apache.http.impl.execchain.ProtocolExec.execute(ProtocolExec.java:184)
	at org.apache.http.impl.execchain.RetryExec.execute(RetryExec.java:88)
	at org.apache.http.impl.execchain.RedirectExec.execute(RedirectExec.java:110)
	at org.apache.http.impl.client.InternalHttpClient.doExecute(InternalHttpClient.java:184)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:71)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:55)
	at org.openqa.selenium.remote.internal.ApacheHttpClient.fallBackExecute(ApacheHttpClient.java:143)
	at org.openqa.selenium.remote.internal.ApacheHttpClient.execute(ApacheHttpClient.java:89)
	at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:142)
	at org.openqa.selenium.firefox.internal.NewProfileExtensionConnection.execute(NewProfileExtensionConnection.java:170)
	at org.openqa.selenium.firefox.FirefoxDriver$LazyCommandExecutor.execute(FirefoxDriver.java:380)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:620)
	... 24 more
Caused by: java.net.ConnectException: Connection refused: connect
	at java.net.DualStackPlainSocketImpl.waitForConnect(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:85)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:339)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:200)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:182)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:579)
	at org.apache.http.conn.socket.PlainConnectionSocketFactory.connectSocket(PlainConnectionSocketFactory.java:74)
	at org.apache.http.impl.conn.DefaultHttpClientConnectionOperator.connect(DefaultHttpClientConnectionOperator.java:134)
	... 39 more
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.remote.UnreachableBrowserException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- webTest.CheckGuestWithValidData -->
    </test> <!-- Check Guest With Valid Data -->
  </suite> <!-- Automation Test -->
</testng-results>
